docs/Manuals/Program/freeciv21-server.rst

```diff
@@ -0,0 +1,175 @@
+
+.. include:: /Manuals/Server/command-line.rst
+    :start-line: 15
+
+
+EXAMPLES
+========
+
+Here are some examples:
+
+freeciv21-server --file oldgame.sav -p 2224
+  Starts a server on port `2224`, loading the savegame named `oldgame.sav`.
+
+freeciv21-server -R ranklog -l logfile -r start-script -f oldgame.sav.xz -p 2224
+  Starts a server on port `2224`, loading the savegame named `oldgame.sav.xz`. Player ranking events are
+  logged to `ranklog`, other logging events are placed in `logfile`. When the server first starts, it
+  immediately executes the server settings script named `start-script`. See freeciv21-server-manual(6) for
+  more information on the server settings script.
+
+freeciv21-server -a -D users.conf -G -N -q 60 -l logfile -s ./saves -p 2224
+  Starts a server on port `2224` with authentication enabled (`-a`, `-D`, `-G`, and `-N`). If no activity
+  for 60 seconds the server will quit (`-q`). Logging events are placed in `logfile` and periodic saves are
+  placed in the `./saves` directory.
+
+
+FILES
+=====
+
+The freeciv21-server requires the default ruleset files to be readable in the Freeciv21 `data` directory.
+This is most often `/usr/local/share/freeciv21`.
+
+* civ2civ3/buldings.ruleset
+* civ2civ3/cities.ruleset
+* civ2civ3/effects.ruleset
+* civ2civ3/game.ruleset
+* civ2civ3/governments.ruleset
+* civ2civ3/nations.rulest
+* civ2civ3/styles.ruleset
+* civ2civ3/techs.ruleset
+* civ2civ3/terrain.ruleset
+* civ2civ3/units.ruleset
+* civ2civ3/script.lua
+* default/default.lua
+* default/nationlist.ruleset
+
+These files are for the default ruleset for the game (civ2civ3), which are loaded if you do not use the
+`--ruleset` arguement. Alternate rules can be loaded with the ``rulsetdir`` directive in a start up script.
+Type ``help resetdir`` at the server command prompt for more information.
+
+
+ENVIRONMENT
+===========
+
+The Freeciv21 server accepts these environment variables:
+
+FREECIV_CAPS
+  A string containing a list of "capabilities" provided by the server. The compiled default should be
+  correct for most purposes, but if you are familiar with the capability string facility in the source you
+  may use it to enforce some constraints between clients and server.
+
+FREECIV_COMPRESSION_LEVEL
+  Sets the compression level for network traffic.
+
+FREECIV_DATA_ENCODING
+  Sets the character encoding used for data files, savegames, and network strings. This should not normally
+  be changed from the default of UTF-8, since that is the format of the supplied rulesets and the standard
+  network protocol.
+
+FREECIV_INTERNAL_ENCODING
+  Sets the character encoding used internally by freeciv21-server. This encoding should not be visible at
+  any interface. Defaults to UTF-8.
+
+FREECIV_LOCAL_ENCODING
+  Sets the local character encoding (used for the command line and terminal output). The default is inferred
+  from other aspects of the environment.
```

Comment:
Don't advertise them. The data and internal encoding should always be UTF-8. The local encoding should be determined by Qt.